src/features/auth/auth.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../styles/mixins.scss';
```

Comment:
Let's use `use` instead of import. Import is deprecated and would be removed in next major realease